integration-tests/src/test/collectcoins.test.ts

```diff
@@ -0,0 +1,64 @@
+import { KeyringPair } from '@polkadot/keyring/types';
+import { AUTHORITY_SURI } from 'creditcoin-js/examples/setup-authority';
+import { createCollectcoinsId } from 'creditcoin-js/extrinsics/request-collectcoins';
+import { createAddressId } from 'creditcoin-js/extrinsics/register-address';
+import { POINT_01_CTC } from '../constants';
+import { Blockchain, creditcoinApi } from 'creditcoin-js';
+import { CreditcoinApi } from 'creditcoin-js/types';
+import { testData } from './common';
+
+describe('CollectCoins', (): void => {
+    let ccApi: CreditcoinApi;
+    let authority: KeyringPair;
+
+    const { keyring } = testData;
+    const evmAddress = '0xffffffffffffffffffffffffffffffffffffffff';
+    const badHash = '0xbad';
+    const blockchain: Blockchain = 'Ethereum';
+    const addressId = createAddressId(blockchain, evmAddress);
+
+    beforeAll(async () => {
+        process.env.NODE_ENV = 'test';
+        ccApi = await creditcoinApi('ws://127.0.0.1:9944');
+        authority = keyring.createFromUri(AUTHORITY_SURI);
+    });
+
+    afterAll(async () => {
+        await ccApi.api.disconnect();
+    });
+
+    it('fail; fee is min 0.01 CTC', async (): Promise<void> => {
+        const { api } = ccApi;
+        const collectcoinsId = createCollectcoinsId(evmAddress);
+        const cause = api.createType('PalletCreditcoinOcwErrorsVerificationFailureCause', 'TaskFailed');
+
+        const { partialFee } = await api.tx.creditcoin
+            .failCollectCoins(collectcoinsId, cause, 1000)
+            .paymentInfo(authority, { nonce: -1 });
+
+        expect(partialFee.toBigInt()).toBeGreaterThanOrEqual(POINT_01_CTC);
+    });
+
+    it('request; fee is min 0.01 CTC', async (): Promise<void> => {
+        const { api } = ccApi;
+
+        const { partialFee } = await api.tx.creditcoin
+            .requestCollectCoins(evmAddress, badHash)
+            .paymentInfo(authority, { nonce: -1 });
```

Comment:
Q: where does `.paymentInfo()` come from ? I don't see the definition.

Reply:
How is the call passing related to the fact that we can't create an authority? As seen in [`sign_and_send`](https://github.com/gluwa/creditcoin-substrate/blob/dev/integration-tests/src/test/fail-transfer.test.ts), extrinsics meant to be called by the OCW require that the caller is an authority or they will early exit with an error. Not being able to sign and send with authority will have extrinsics bounce at "expect(authority)", is there a way to make them pass?

Reply:
Also per [the docs](https://polkadot.js.org/docs/api/cookbook/tx/) and [`paymentInfo`](https://github.com/polkadot-js/api/blob/master/packages/api/src/submittable/createClass.ts#L118) it looks that It would actually be easier to fake the caller with `paymentInfo` than `sign_and_send`.